src/components/LoadingSpinner.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import { css } from 'glamor';
+
 import { LoadingIcon } from '../components/SvgIcons';
 
 export default function LoadingSpinner() {
   return (
-    <div className="loading-spinner">
+    <div
+      className="loading-spinner"
+      {...css({
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100%',
+      })}
+    >
       <LoadingIcon className="loading-spinner-svg" />
```

Comment:
This is a nice example of a pre-styled, reusable component! 👍

It would also be nice to remove the no-longer-necessary `loading-spinner` and `loading-spinner-svg` class names as part of this style cleanup. 😄